src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -854,7 +854,7 @@ FixedwingPositionControl::control_position(const Vector2f &curr_pos, const Vecto
 			float loiter_radius = pos_sp_curr.loiter_radius;
 			uint8_t loiter_direction = pos_sp_curr.loiter_direction;
 
-			if (pos_sp_curr.loiter_radius < 0.01f || pos_sp_curr.loiter_radius > -0.01f) {
+			if (pos_sp_curr.loiter_radius < 0.01f && pos_sp_curr.loiter_radius > -0.01f) {
```

Comment:
How about checking the absolute value instead?

Reply:
Makes sense. I also added FLT_EPSILON instead of 0.01f, which makes the intent more clear.